RELEASE.md

```diff
@@ -39,6 +39,84 @@ See the next section for details on cutting an individual release.
 
 ## How to cut an individual release
 
+### Release issue template
+
+The quick and easy way to create a GitHub issue using the following template and follow the instructions from the issue itself.
+If something is not clear, you can get back to this document to learn more about the process.
+
+````markdown
+### Publish the release candidate
+
+- [ ] Wait for any open PR we want to get merged before cutting the release candidate
+  - [ ] Eventually open a PR for every experimental feature we want to promote to stable
+  - [ ] Eventually open a PR to remove any deprecated feature or configuration option that should be removed in this release
+- [ ] Update `CHANGELOG.md`
+  - [ ] Run `./tools/release/check-changelog.sh LAST-RELEASE-TAG...main` and add missing PRs to CHANGELOG
+  - [ ] Ensure CHANGELOG entries are [sorted by type](https://github.com/grafana/mimir/blob/main/docs/internal/contributing/README.md#changelog)
+  - [ ] Add a new section for the new release so that `## main / unreleased` is blank and at the top. The new section should say `## x.y.0-rc.0`.
+- [ ] Run `./tools/release/notify-changelog-cut.sh`
+- [ ] Run `make mixin-screenshots`
+  - Before opening the PR, review all updated screenshots and ensure no sensitive data is disclosed
+- [ ] Create new release branch
+  ```bash
+  git checkout main
+  git checkout -b release-<version>
+  git push -u origin release-<version>
+  ```
+- [ ] Publish the Mimir release candidate
+  - [ ] Begin drafting the [release notes](https://github.com/grafana/mimir/blob/main/RELEASE.md#write-release-notes-document), but don't block on it to publish the release candidate
```

Comment:
This has nothing to do with "Publish the Mimir RC", and can be moved to earlier step. It should perhaps mention that this can be done on `main` branch, and final release note is later cherry-picked into the branch.